as/cNFT/assembly/index.ts

```diff
@@ -1,13 +1,28 @@
-import { persistent_tokens } from './models/persistent_tokens'
+import { storage } from 'near-sdk-as'
```

Comment:
As I see `storage` is not used anywhere inside the file. So is it necessary to include here?

Reply:
:) cleaned